README.md

```diff
@@ -108,6 +111,16 @@ python src/parse_json_for_plots.py
 
 This will create and save plots as .png files under `data/plots`.
 
+## Development dependencies
```

Comment:
Is there a use case for the instructions in this section? My inclination would be to always use `tox` for managing the benchmark environments, so we don't need to know how to install these extras directly using pip.

Reply:
👍 that sounds like a good use case to me